Documentation/project-docs/platform-names.md

```diff
@@ -0,0 +1,39 @@
+# Platform Names
+
+Over the years, we've accumulated quite a few names, platforms, and components that can make it hard for folks to understand what we're referring to. This document has a list that will help to qualify what we mean by what.
+
+This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
+
+## .NET Framework
+
+**Also referred to as**: Desktop, full framework, in-box framework
+
+This refers to the .NET Framework that shipped in 2002. It's the main framework folks target today and allows you to build a wide variety of applications, such as WinForms, WPF, ASP.NET, and command line tools.
+
+## .NET Core
+
+**Also referred to as**: UWP, Store
+
+Originally, .NET Core was the identifier we used to describe the surface area Windows 8 store applications can use. This may seem odd, but when we designed the API set, we wanted to created a foundation for .NET where portability is a first class concern for the layering and componentization. For more details, you may want to read [this blog post](http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx).
+
+Today, .NET Core no longer stands for store applications alone -- .NET Core is the name for the open source, cross-platform .NET stack that supports ASP.NET as well as UWP applications. The stack includes a set of framework libraries (CoreFX), a JIT based runtime (CoreCLR), an AOT based runtime (CoreRT), and a set of tooling (such as the dotnet CLI).
+
+So you can think of today's .NET Core as an evolution of the original surface area we used for store applications. Many of the original design goals are still relevant, especially around layering and portability.
+
+## Universal Windows Platform (UWP)
+
+**Also referred to as**: Store, WinRT, Metro
+
+The Universal Windows Platform (UWP) is the platform that is used for building modern, touch-enabled Windows applications. It's designed to unify the different types of devices that you may want to target, including PCs, tablets, phablets, phones, and even the Xbox.
+
+UWP provides many services, such as a centralized app store, an execution environment (AppContainer), and a set of Windows runtime APIs (WinRT). While UWP itself doesn't provide .NET APIs, the Visual Studio tooling experience will present UWP as a platform that is composed of both, WinRT APIs, as well as .NET APIs. The .NET side is represented by .NET Core.
+
+## .NET Native
```

Comment:
You don't really explain the major difference between NGEN and .NET Native: That Native outputs a single exe with all dependences internal while NGEN does not.